### ISSUE TITLE
[RESOLVED] Server won't start anymore - [BetterFps Update Checker/WARN] [BetterFps]: Could not check for updates: widget.mcf.li

### ISSUE THREAD

**Issue by JustFlaaffy** [DarkPacks/SevTech-Ages#4846]
## Bug Description

I've been running a SevTech Ages server for a couple weeks now, but turned my computer off last night to install a new desk in my office. My computer forced a Windows update and now my SevTech Ages server won't work anymore. I've tried to uninstall the Windows updates it installed, but I'm still unable to get the server to function. The last line in the CMD Prompt window is: [BetterFps Update Checker/WARN] [BetterFps]: Could not check for updates: widget.mcf.li

### Did this behavior use to work in the previous version?

I haven't changed any of the files for the server, the only thing that's changed is that Windows forced an update. I have a friend who was trying to run a SevTech Ages server for me but wasn't able to get it to work, but it worked on my machine, hence why I'm running the server.

## Steps to Reproduce (for bugs)

1. Restart machine
2. Windows forced its horrible updates
3. Server won't start :(

## Logs

* Client/Server Log:
server log is in two parts because pastebin has a size limit
Part 1: [https://pastebin.com/p1mbNshm](https://pastebin.com/p1mbNshm)
Part 2: [https://pastebin.com/g2rQWSr0](https://pastebin.com/g2rQWSr0)
* Crash Log:
* there is no crash report, it's not crashing. just won't start properly anymore

## Client Information
* Modpack Version:
* SevTech Ages 3.2.3
* Java Version:
* 1.8 Update 351
* Launcher Used:
* Curseforge
* Memory Allocated:
* about 20 GB
* Server/LAN/Single Player:
* Server
* Optifine Installed:
* Yes
* Shaders Enabled:
* client side only

## World Information
* Modpack Version world created in:
* 3.2.3
* Additional Content Installed:
* no

## Server Information
* Java Version:
* 1.8 Update 351
* Operating System:
* Windows 10
* Hoster/Hosting Solution:
* I'm unsure what this means
* Sponge (Non-Vanilla Forge) Server:
* I'm unsure what this means

**Comment by sam-kirby** [comment]
The server looks to have started correctly from the logs; what issue are you seeing when trying to connect?

It's possible that your IP address changed when you rebooted, have you checked the settings you're using to connect?

**Comment by JustFlaaffy** [comment]
> The server looks to have started correctly from the logs; what issue are you seeing when trying to connect?
> 
> It's possible that your IP address changed when you rebooted, have you checked the settings you're using to connect?

I ended up messing with this for hours last night and fixed it. When I restarted my computer, I was given an external ip like 2.x.x.x, which i thought was a bit weird. I was able to get it to connect locally (on the same machine) using ipv4, but whenever I'd use the external ip address with port number it wouldn't work. So what I did was connected to nord vpn to get my external ip to change, which didn't work. BUT when I disconnected from nord vpn, my external ip went back to the ip address I had previously that worked. I also changed one Firewall setting in Windows: "Inbound connections that do not match a rule are blocked." in Windows Defender Firewall with Advanced Security to "allowed"
After that, it's started working again. I'm definitely not super savvy with networking and firewalls, but I do work in IT. So I'm proud of myself for figuring that out, but boy it was annoying.
